packages/e2e-tests/src/specs/editor/inspector.js

```diff
@@ -20,24 +20,24 @@ import { createNewStory, withUser } from '@web-stories-wp/e2e-test-utils';
 
 describe('Inspector', () => {
   describe('Document', () => {
-    it('should not display publisher logo', async () => {
+    it('should display publisher logo upload', async () => {
       await createNewStory();
 
       await expect(page).toClick('li[role="tab"]', { text: 'Document' });
-      await await expect(page).toMatchElement('[aria-label="Publisher Logo"]');
+      await expect(page).toMatchElement('[aria-label="Publisher Logo"]');
     });
     describe('Contributor User', () => {
       withUser('contributor', 'password');
 
-      it('should not display publisher logo', async () => {
+      // eslint-disable-next-line jest/no-disabled-tests
+      it.skip('should not display publisher logo upload', async () => {
```

Comment:
Why is this now skipped?

Reply:
I'm just testing what's going on here (_"Test fail reason"_).
Locally I'm getting this, any ideas?

![image](https://user-images.githubusercontent.com/21036224/134068234-e84ee647-57e8-47f4-9282-a3d9edb1275c.png)

Reply:
The easiest way to find out is by opening the editor as a contributor user and then checking the browser console.
The more complicated way would be by running your tests in non-headless mode and observing the behavior.

Currently, they contributors get a 403 because they don't have access to the `web-stories/v1/settings` route.

You need to check for `canManageSettings` capabilities before trying to access it.

As per our lengthy discussion above, I opened #9007 to move publisher logos to a separate REST route that's open for contributors. I might get to that tomorrow if Jonny doesn't before me.

Until then, you'd need to work around it.

Reply:
Happy to chat about it tomorrow if things are unclear.